src/components/autocomplete/Autocomplete.vue

```diff
@@ -308,6 +310,12 @@ export default {
             this.$emit('input', value)
             // Check if selected is invalid
             const currentValue = this.getValue(this.selected)
+
+            // Fixes #3185
+            if (!currentValue) {
```

Comment:
Better a typeof undefined check and an else if ;)

Reply:
Hi @jtommy I have been busy with the release of a project :( but I will have a look tomorrow and submit a commit.

By the way, I found that strict type checking has some bugs with specific minifier settings, for example google closure compiler and terser with the option of booleans as integers will trigger errors with Buefy because of the strict type checking. Everything else runs as expected.